ui/app/templates/components/allocation-row.hbs

```diff
@@ -11,7 +11,7 @@
       role="tooltip"
       aria-label="Allocation depends on unhealthy drivers"
     >
-      {{x-icon "alert-triangle" class="is-warning"}}
+      <FlightIcon @name="alert-triangle-fill" @color="#fa8e23" />
```

Comment:
General pattern: bulma classes to add colour vs expressly using the @color prop.